BUILD

```diff
@@ -34,7 +35,6 @@ cc_library(
         "@boringssl//:crypto",
         "@com_google_protobuf//:protobuf_lite",
         "@proxy_wasm_cpp_sdk//:api_lib",
-        "@wamr//:wamr_lib",
-        "@wasm_c_api//:wasmtime_lib",
```

Comment:
why remove wasmtime_lib?

Reply:
keep one WASMVM to avoid unnecessary linking problems. Both @wamr and @wasm_c_api actually implement wasm_c_api, which means they have duplicate implementation for the same APIs.